eberron.md

```diff
@@ -38,7 +38,7 @@ Automatons are both a great mystery and tightly kept secret. There have been rum
 Modern automatons are built in a wide array of configurations but they are typically large bi-pedal entities that resemble humanoid creatures.
 
 - **Starting Wounds:** 3
-- **Move/Sprint:** 3/4
+- **Move/Sprint:** 3/2
```

Comment:
4 is a very high sprint, with someone getting dex able to reach 9-10 in just sprint.